src/vtquery.cpp

```diff
@@ -270,15 +265,17 @@ struct Worker : Nan::AsyncWorker {
                                 properties_map.insert(std::pair<std::string, variant_type>(key, value));
                             }
 
-                            ResultObject r(ll, meters, properties_map, layer_name, geom_type);
-                            hits.push_back(r);
+                            ResultObject r(feature_lnglat, meters, properties_map, layer_name, geom_type);
+                            results_.push_back(r);
```

Comment:
It might be more efficient here to use an `emplace_back` rather then a `push_back` as you are not using the `ResultObject` after creating it.

Reply:
Good call 👍